### ISSUE TITLE
order of external communities on community page

### ISSUE THREAD

**Issue by alexmitrani** [climu/openstudyroom#320]
Ideally they should be ordered from top to bottom with the most active communities at the top and the least active communities at the bottom.  
Another approach would be to verify with the inactive communities if they plan to resume activities, and if not, they could be removed.  
thanks.

**Comment by smashbits** [comment]
Currently the activity-check could be done manually since the number of communities is fairly small. OSR-admins could get in contact with community-owners/admins to check if the community is still used/needed. If not there should be a flag to set the community as inactive which would hide it. Such flag could also be used for setting up new communities before making them visible to the public.

**Comment by climu** [comment]
Yes, we already have a flag called `public` that will hide the community to anyone but it's members.

We can use that for now to manage the inactive leagues.

This doesn't really address the coding issue regarding ordering of the communities.
